docs/requirements.md

```diff
@@ -2,88 +2,76 @@
 
-```text
-CPU: workerResources.requests.cpu + (pluginResources.requests.cpu * 4)
+| Installation type                        | ~ Total vCPUs | ~ Total Memory | Replicas per component | Max. commits analyzed concurrently | Platform vCPUs | Platform Memory | Analysis Workers vCPUs | Analysis Workers Memory |
+| ---------------------------------------- | ------------- | -------------- | ---------------------- | ---------------------------------- | -------------- | --------------- | ---------------------- | ----------------------- |
+| Kubernetes Small Installation            | 16            | 32 GB          | 1                      | 2                                  | 4              | 8 GB            | 10                     | 20 GB                   |
+| Kubernetes Medium Installation (default) | 32            | 64 GB          | 2                      | 4                                  | 8              | 16 GB           | 20                     | 40 GB                   |
+| Kubernetes Big Installation              | 60+           | 110+ GB        | 2+                     | 10+                                | 8+             | 16+ GB          | 50+                    | 100+ GB                 |
 
-Memory: workerResources.requests.memory + (pluginResources.requests.memory * 4)
+**NOTE:**
+For microk8s we added 1.5 CPU and 1.5 GB extra in the "Platform" meant to be used by microk8s itself.
 
-Number of concurrent analysis: workers.dedicatedMax
-```
+| Installation type              | ~ Total vCPUs | ~ Total Memory | Replicas per component | Max. commits analyzed concurrently | Platform vCPUs | Platform Memory | Analysis Workers vCPUs | Analysis Workers Memory |
+| ------------------------------ | ------------- | -------------- | ---------------------- | ---------------------------------- | -------------- | --------------- | ---------------------- | ----------------------- |
+| MicroK8s Minimum               | 16            | 32 GB          | 1                      | 2                                  | 5.5            | 9.5 GB          | 10                     | 20 GB                   |
+| MicroK8s Recommended (default) | 20+           | 32+ GB         | 1-2                    | 2                                  | 11+            | 20+ GB          | 10                     | 20 GB                   |
 
-Given the previous values, the total number of resources required should be the "per-analysis" amount times the number of concurrent analysis.
+### Storage
 
-### Example
+The storage necessary also depends on the number of repositories you have.
+You can use the following table as a guideline to understand the kind of storage allocation
+done by codacy.
 
-_Maximum of 6 concurrent analysis_
+| Installation type | Bundled in the chart         | Minimum Recommended |
```

Comment:
"Installation type" doesn't seem to apply to the contents of the column. :thinking: Could it be something like "Component"?

```suggestion
| Component | Bundled in the chart         | Minimum Recommended |
```